### PR TITLE
Add feature list to readme

### PR DESCRIPTION
**Pull request by georgius1024**
- Added todo
- Added install instructions
- added feature list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 ### otus-qa-2022-04
 This file is edited twice
+
+### Feature list 
+* 1
```
Comment (georgius1024) [inline README.md:4]: Why did you....

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+* 2
+* 3
+
+### Installation instructions
+Lorem ipsum lalala
```
Comment (georgius1024) [inline README.md:9]: Why didn't you....

**Comment by Takaraka** [comment]
ok

File: README.md (hunk 1 above)
Comment (georgius1024) [reply README.md:4]: Because....

**Comment by georgius1024** [comment]
Tested, works as expected

**Pull request merged**